src/main/java/uk/gov/hmcts/reform/sscs/domain/wrapper/AnswerState.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.domain.wrapper;
+
+public enum AnswerState {
```

Comment:
Don't think this is needed anymore?

Reply:
OnlineHearing data object is used by both MYA and COR, can't removed it unless if we modify the frontend